data/ShreyashSomvanshi.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Shreyash Somvanshi",
+  "type": "personal",
+  "bio": "Artificial Intelligence & Data Science Engineer | Open Source Beginner | Technical Content Writer ",
+  "tags": ["Data Science", "AI", "Open Source", "Content Writer"],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/ShreyashSomvanshi" },
+    { "icon": "SiMedium", "url": "https://medium.com/@ShreyashSomvanshi" }
+  ],
+  "testimonials": ["None...."],
```

Comment:
It would be better to remove this section if you currently don't have a testimonial. 
```suggestion
```

Reply:
Updated! @kumarsonsoff3